ros2cli/ros2cli/helpers.py

```diff
@@ -133,3 +136,61 @@ def get_rmw_additional_env(rmw_implementation: str) -> Dict[str, str]:
         return {
             'RMW_IMPLEMENTATION': rmw_implementation,
         }
+
+
+def interactive_select(
+    items: List[str],
+    prompt: str = 'Select an item:'
+) -> Optional[str]:
+    """
+    Launch interactive fuzzy search using fzf to select from a list of items.
+
+    :param items: List of items to select from
+    :param prompt: Prompt message to display in fzf
+    :return: Selected item or None if user cancelled or fzf not available
+    """
+    if not items:
+        print('No items available to select from.', file=sys.stderr)
+        return None
+
+    try:
+        # Check if fzf is available
+        result = subprocess.run(
+            ['fzf', '--version'],
+            capture_output=True,
+            text=True,
+            timeout=1
+        )
+        if result.returncode != 0:
+            raise FileNotFoundError()
+    except (FileNotFoundError, subprocess.TimeoutExpired):
+        print(
+            'Error: fzf is not installed but is a dependency for this package. You can install it with rosdep',
+            file=sys.stderr
+        )
+        return None
+
+    try:
+        # Launch fzf with items as input - using direct TTY access
+        process = subprocess.Popen(
+            ['fzf', '--prompt', prompt + ' ', '--height', '40%', '--reverse'],
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            text=True
+        )
+
+        # Send items to fzf
+        stdout, _ = process.communicate(input='\n'.join(items))
+
+        # Check if user cancelled (Ctrl-C or ESC)
+        if process.returncode != 0:
+            return None
+
+        # Return selected item (strip newline)
+        selected = stdout.strip()
+        return selected if selected else None
+
+    except Exception as e:
+        print(f'Error during interactive selection: {e}', file=sys.stderr)
+        return None
```

Comment:
Catching all exceptions hides bugs and makes debugging harder.


```suggestion
    except (OSError, subprocess.SubprocessError) as e:
        print(f'Error during interactive selection: {e}', file=sys.stderr)
        return None
```